v0/destinations/bqstream/transform.js

```diff
@@ -1,45 +1,17 @@
 /* eslint-disable no-console */
 const { EventType } = require("../../../constants");
-const {
-  defaultRequestConfig,
-  defaultPostRequestConfig,
-  CustomError,
-  getSuccessRespEvents,
-  getErrorRespEvents,
-  isOAuthDestination
-} = require("../../util");
+const { CustomError } = require("../../util");
 
-/**
- * This function puts the accessToken information in the transformed response
- * to facilitate for a successfully authorised event
- *
- * @param {*} event - The event on which transformation is being performed
- * @param {*} response - Transformation Response
- * @returns Transformation Response bound with token information(may not be required as such)
- */
-async function processAuth(event, response) {
-  // OAuth for BQStream destination
-  const { oauthAccessToken } = event.metadata;
-  if (!oauthAccessToken) {
-    throw new CustomError("Access token is empty/undefined", 500);
+const getInsertId = (properties, insertId) => {
+  if (
+    properties[insertId] &&
+    (typeof insertId === "string" || typeof insertId === "number")
```

Comment:
since `insertId` here is denoting one of the columns (or in other words one of the property fields), should we keep the `number` check ?

Reply:
We are just being explicit wrt what can be stringified and what can't be stringified.